querybook/webapp/components/DataDocRightSidebar/ScheduleDataDocButton.tsx

```diff
@@ -3,20 +3,49 @@ import { DataDocSchedule } from 'components/DataDocSchedule/DataDocSchedule';
 
 import { IconButton } from 'ui/Button/IconButton';
 import { Modal } from 'ui/Modal/Modal';
+import { Tabs } from 'ui/Tabs/Tabs';
+import { InfoButton } from 'ui/Button/InfoButton';
 
 interface IProps {
     docId: number;
     isEditable: boolean;
 }
 
+const scheduleTabs = ['schedule', 'history'];
+
+export type IScheduleTabs = typeof scheduleTabs[number];
+
 export const ScheduleDataDocButton: React.FunctionComponent<IProps> = ({
     docId,
     isEditable,
 }) => {
     const [showForm, setShowForm] = React.useState(false);
-    const modal = showForm && (
-        <Modal onHide={() => setShowForm(false)} title="Schedule DataDoc">
-            <DataDocSchedule isEditable={isEditable} docId={docId} />
+
+    const [currentTab, setCurrentTab] = React.useState<IScheduleTabs>(
+        'schedule'
+    );
+
+    const topDOM = (
```

Comment:
the topDOM should be enclosed as part of DataDocSchedule, if thats an issue, consider creating a new component called DataDocScheduleModal. The button itself should only concern with the visibility of the modal